Makefile

```diff
@@ -226,6 +226,12 @@ deploy-role: deploy-namespace
 deploy-postgres: deploy-namespace
 	python3 ./tools/deploy_postgres.py --namespace "$(NAMESPACE)" --profile "$(PROFILE)" --target "$(TARGET)"
 
+deploy-service-on-cluster:
+	export TARGET=day2 && $(MAKE) deploy-postgres deploy-ocm-secret deploy-s3-secret deploy-service
```

Comment:
let's rename target to "ocp" and not "day2". "day2" currently will be used for day2 operations on cloud

Reply:
done